python/lib/sift_grafana/sift_query_model.py

```diff
@@ -0,0 +1,56 @@
+from typing import List, Optional
+
+from pydantic import BaseModel, Field, model_validator
+from typing_extensions import Self
+
+
+# These are the models that are used to validate the input data for the Sift Grafana Plugin queries
+class Channel(BaseModel):
+    channelId: Optional[str] = None # noqa: N815
+    channelIdentifier: Optional[str] = None # noqa: N815
+
+    @model_validator(mode='after')
+    def check_passwords_match(self) -> Self:
+        channel_id = self.channelId
+        channel_identifier = self.channelIdentifier
```

Comment:
Not sure if there's a particular reason for camelCase, but if there isn't a particular requirement could we keep vars as snake_case?

Reply:
These variables match the json in the schema, which are camelCase. I've tried to keep everything that was python only to the snake case